supabase/migrations/20240530291033_add_completed_parcels_view.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW completed_parcels AS
+WITH completed_events AS (
+    SELECT
+        events.parcel_id,
+        MAX(events.timestamp) AS completed_timestamp
```

Comment:
See below comment :) -> So we don't double count parcels if they are set to multiple "successful" states, there's nothing in the frontend stopping this happening and I would have though it may happen by accident a few times. MAX(timestamp) is just an aggregate function - if there are two successful states, we pick the latest.     Not sure how important this is though, happy to remove if the simplicity is more important!

Reply:
Nope that makes sense let's leave, but if possible maybe add a comment explaining it